tokio/CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+# 1.8.0 (July 2, 2021)
+
+### Added
+
+- io: add `get_{ref,mut}` methods to `AsyncFdReadyGuard` and `AsyncFdReadyMutGuard` ([#3807])
+- io: efficient implementation of vectored writes for `BufWriter` ([#3163])
+- net: add ready/try methods to `NamedPipe{Client,Server}` ([#3866], [#3899])
+- rt: add `tokio::task::Builder` ([#3881])
```

Comment:
This is an unstable API, so it will not be available to most users. We probably either want to split it out to a separate section or just not include it in the changelog.

Reply:
yeah, i think an "Unstable APIs" section might be worth adding?